bash/lsst/sitcom/2023/package_setup-obsrun-2023-03B.sh

```diff
@@ -0,0 +1,226 @@
+#!/usr/bin/env bash
+# This script should be executed (not sourced!) from a Nublado terminal
+# to clone and setup the packages needed to configure the
+# Nublado environment to use methods from
+# rapid_analysis, such as bestEffortISR
+# it's required to run the latiss_align_and_take_sequence
+# and the latiss_cwfs_align scripts
+
+# set to exit when any command fails
+set -e
+
+# This script is ONLY to be used for the AuxTel run of 2023-01B
```

Comment:
date change here...
might be best to just remove this comment actually. The filename has it..